### DISCUSSION TITLE
Pods are missing from UI but only for deployments

### DISCUSSION THREAD

**Discussion by MrLuje** [argoproj/argo-cd#11845] [Q&A] [answered]
We have been using ArgoCD on a Rancher cluster for a few years without any issues.

Now we are trying to use the same stack on a openstack k8s cluster.

This is working mostly fine except we are missing deployment's **pods**.
After checking the cluster, this deployment has 1 pod but is completely missing from UI
![image](https://user-images.githubusercontent.com/632075/209803710-247ca781-7dd4-4492-8336-d71a72639d5d.png)

I checked other resources and it seems all deployments are not showing pods but cronjobs & statefulsets are properly showing
![image](https://user-images.githubusercontent.com/632075/209804996-ee8a4927-2051-4bde-9868-da8decdfc93f.png)

ArgoCD (v2.4.12) is installed by helm with **rbac.create** flag

Maybe something related to replicaset permissions ?
Not totally sure where to start investigating...

Thanks in advance for having a look 🙏

**Comment by jannfis** [comment]
Any chance your two applications are in different AppProjects?

If yes, please check if the AppProject for `datadispatcher-broker-dev` actually allows `Pod` kind in the resource list. The web UI will not show resources that are not allowed in the AppProject for a given application.

**Reply by MrLuje** [reply]
For these 2, yes... there are from different projects but are setup using the same access template; but I also have the same behavior from Deployment/Statefulset within the same project
The weird thing is ArgoCD **can** deploy pods without issue, the only issue is the missing items in UI

**Comment by tanerjn** [comment]
why not Jenkins ?

**Comment by MrLuje** [comment, marked as answer]
We found the issue.
We created these projects with reduced namespaced ressources permissions, "Pods" where properly added but not "apps/ReplicaSet"
So we indirectly had permission to deploy them through "Pods" but not to view them; since Pods are inherited from ReplicatSet, both were missing

**Reply by mikecarr** [reply]
Do you have a sample of what your AppProject looks like? I assume this is where you added apps/ReplicaSet. I have the same issue and adding did not solve it
```
    - group: 'apps'
      kind: 'ReplicatSet'
    - group: ''
      kind: 'Pod'

```

**Reply by ThSchrad** [reply]
If you copy/pasted the section of your AppProject, it should be `ReplicaSet` instead of `ReplicatSet`.
